### ISSUE TITLE
Handling empty ("") next-link in PagedList

### ISSUE THREAD

**Issue by anuchandy** [Azure/azure-sdk-for-java#1641]
PagedList invokes callback to load next page only if `Page.nextLink` is not NULL, but some services seems returning nextPage link as "" to indicate no more page follows (instead of  NULL). Given below an example. Question is should we treat NULL and "" as same in our PagedList or is it a bug in the service? Today with nextLink = "", generated code try to do a GET on base URI and call fails.

```json
{
  "value": [
    {
      "name": "Microsoft.ServiceBus/checkNameAvailability/action",
      "display": {
        "provider": "Microsoft Azure ServiceBus",
        "resource": "Non Resource Operation",
        "operation": "Get namespace availability.",
        "description": "Checks availability of namespace under given subscription."
      },
      "properties": null
    },
    {
      "name": "Microsoft.ServiceBus/register/action",
      "display": {
        "provider": "Microsoft Azure ServiceBus",
        "resource": "ServiceBus Resource Provider",
        "operation": "Registers the ServiceBus Resource Provider",
        "description": "Registers the subscription for the ServiceBus resource provider and enables the creation of ServiceBus resources"
      },
      "properties": null
    },
    {
      "name": "Microsoft.ServiceBus/namespaces/write",
      "display": {
        "provider": "Microsoft Azure ServiceBus",
        "resource": "Namespace",
        "operation": "Create Or Update Namespace ",
        "description": "Create a Namespace Resource and Update its properties. Tags and status of the Namespace are the properties which can be updated."
      },
      "properties": null
    },
    ...
  ],
  "nextLink": ""
}
```

**Comment by anuchandy** [comment]
Addressed this issue in this [commit](https://github.com/Azure/autorest-clientruntime-for-java/pull/258/commits/6afeaf08883db5438f1ac050ddb1f3d77894e1e5)

**Comment by martinsawicki** [comment]
fixed already in runtime v1.1.1
